hs_core/hydroshare/resource.py

```diff
@@ -22,6 +22,10 @@
 from hs_core.hydroshare import utils
 from hs_access_control.models import ResourceAccess, UserResourcePrivilege, PrivilegeCodes
 from hs_labels.models import ResourceLabels
+from django.core.exceptions import ImproperlyConfigured
+from haystack import connections, connection_router, constants
+from haystack.backends.solr_backend import SolrSearchBackend
+
```

Comment:
@xuezhaokun I believe the three import lines above are left-over and should be removed, since they are not used in the module.